pkg/template/sops.go

```diff
@@ -143,7 +143,7 @@ func (t *Sops) decryptSopsFile(path, newPath string, args, env []string) error {
 
 	err := cmd.Run()
 	if err != nil {
-		return fmt.Errorf("Running sops: %s", err)
+		return fmt.Errorf("Running sops: %s, %v", stderrBs.String(), err)
```

Comment:
Would we still want to include `err` here? 

Do you have an example of what this looks like when an error occurs?

Reply:
Yes. Before this change, the error looks like:

```
Running sops: exit code 1
```

After the change, it looks something like:

```
Running sops: Failed to get the data key required to decrypt the SOPS file.
Group 0: FAILED
  0979BA19CE2BAEF3: FAILED
    - | could not decrypt data key with PGP key:
      | golang.org/x/crypto/openpgp error: Reading PGP message
      | failed: EOF; GPG binary error: exec: "gpg": executable file
      | not found in $PATH
Recovery failed because no master key was able to decrypt the file. In
order for SOPS to recover the file, at least one key has to be successful,
but none were. exit status 128
```

Reply:
Ah my mistake. I didn't see the conversion to `%v`. Thanks and lgtm.